frontend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Base and builder image will need to be replaced by Fips compliant one
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9-1161 AS builder
+RUN microdnf install -y tar make git
```

Comment:
Notably switched the builder container to ubi-minimal as well and am installing git into the builder container